Project.toml

```
@@ -35,14 +35,13 @@ Tracker = "9f7883ad-71c0-57eb-9f7f-b5c9e6d3789c"
 AbstractMCMC = "0.5.2"
 AdvancedHMC = "0.2.20"
 AdvancedMH = "0.4"
-Bijectors = "0.6.2"
+Bijectors = "~0.6.2"
```

That's the same as
```suggestion
Bijectors = "0.6.2"
```
BTW, so the change is not needed.

---

I am fine with both. I added the ~ because of Will's comment above. I prefer the simpler version though so happy to bring it back.

---

See https://julialang.github.io/Pkg.jl/v1/compatibility/#Version-specifier-format-1.